tools/setup_helpers/generate_code.py

```diff
@@ -48,7 +48,11 @@ def generate_code(ninja_global=None,
             'tools/autograd',
             disable_autograd=disable_autograd,
         )
-        gen_jit_dispatch(declarations_path or DECLARATIONS_PATH, jit_gen_dir, 'tools/jit/templates')
+        gen_jit_dispatch(
+            declarations_path or DECLARATIONS_PATH,
+            jit_gen_dir,
+            'tools/jit/templates',
+            disable_autograd=disable_autograd)
```

Comment:
I think technically backward ops are not coupled with autograd? Maybe I should rename all "disable_autograd" to "inference_only"? Just don't want to create too many different concepts while the main purpose is just to shrink prebuilt mobile lib size...

Reply:
I think the same setting makes sense - and the backward ops ARE coupled with autograd. They are not exposed to python, so the only way to use them is through autograd